packages/babel-parser/src/types.js

```diff
@@ -387,7 +387,7 @@ export type YieldExpression = NodeBase & {
 
 export type AwaitExpression = NodeBase & {
   type: "AwaitExpression",
-  argument: ?Expression,
+  argument: Expression,
```

Comment:
While `yield` is a yield expression, `await` is never valid and thus the `argument` here can not be `null`.